pkg/operator/operator.go

```diff
@@ -69,6 +70,9 @@ func NewOpenShiftControllerManagerOperator(
 	// we only watch some namespaces
 	kubeInformersForOpenshiftControllerManager.Core().V1().Namespaces().Informer().AddEventHandler(c.namespaceEventHandler())
 
+	// set this bit so the library-go code employed for clusteroperator reconciliation allows this operator to be set to unmanaged
```

Comment:
```suggestion
	// set this bit so the library-go code employed for clusteroperator reconciliation does not allow this operator to be set to unmanaged
```

Reply:
no ... it is allows, not "does not allow"